library/components/TTextButton.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <button class="
```

Comment:
BaseButton kept on compiling to `basebutton` in the html which has just a `div` with a fancy name. So, to avoid struggling with it I used `button`.

Reply:
Not worth holding up merging, but did you try basse-button? I’m wondering if the back end isn’t doing something odd when building.

Reply:
I'll give it another whirl and see.

Reply:
It works! Must have just been having a moment when I was moving compnents about. I removed the props and the export default since it only relies on a slot.